.github/workflows/e2e.v2.yaml

```
@@ -0,0 +1,193 @@
+        mode: [stream, unary, multi]
+        deployment: [helm-chart, helm-operator]
+        environment: [readreplica] # [null, management, mirror, readreplica]
+        # TODO: Chaos 見直し
+        # chaos: [null, failure, kill, partition, bandwidth]
+        include:
+          - mode: stream
+            deployment: helm-chart
+            environment: profile
+        exclude:
+          - deployment: helm-operator
+            environment: management
```

actionlint (1.7.4)

value "management" in "exclude" does not match in matrix "environment" combinations. possible values are "readreplica", "profile"

(matrix)